presto/presto_test.go

```diff
@@ -238,6 +238,41 @@ func TestQueryWithJustUserHeader(t *testing.T) {
 	}
 }
 
+func TestQueryForUsername(t *testing.T) {
```

Comment:
I guess you can just change the `TestQueryWithJustUserHeader` to query for `current_user` and have and rename it to `TestQueryForUsername`; but having two tests is fine too.
Please squash two commits together.

Reply:
Two tests seems to overlap each other, so I got rid of `TestQueryWithJustUserHeader` because `TestQueryForUsername` tests with just user header.